src/tribol/geom/GeomUtilities.hpp

```diff
@@ -149,15 +150,15 @@ void Local2DToGlobalCoords( real xloc, real yloc,
  * \note this assumes that the point lies in the plane defined by the 
  *  2D local basis vectors. 
  */
-void GlobalTo2DLocalCoords( const real* const RESTRICT pX, 
-                            const real* const RESTRICT pY, 
-                            const real* const RESTRICT pZ,
-                            real e1X, real e1Y, real e1Z,
-                            real e2X, real e2Y, real e2Z,
-                            real cX, real cY, real cZ,
-                            real* const RESTRICT pLX, 
-                            real* const RESTRICT pLY, 
-                            int size );
+TRIBOL_HOST_DEVICE bool GlobalTo2DLocalCoords( const RealT* const pX, 
```

Comment:
Rationale behind changing return type from void to bool?

Reply:
Looks like this is related to error handling, but isn't fully implemented.  Reverted.